schema-util/json/src/main/java/io/apicurio/registry/rules/SomeJsonSchema.java

```diff
@@ -0,0 +1,53 @@
+package io.apicurio.registry.rules;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.erosb.jsonsKema.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import static io.apicurio.registry.rules.validity.JsonSchemaVersion.DRAFT_2020_12;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Wraps a parsed JSON schema, that can be represented by exactly one of two different implementations.
+ */
+@ToString
+@EqualsAndHashCode
+public class SomeJsonSchema {
```

Comment:
I hate writing this kind of comment, but, is there a particular reason for this weird name?

Reply:
Didn't come up with a better one🤷‍♂️ Any suggestions?

Reply:
Renamed to `ParsedSchema`.